fms/models/llama.py

```diff
@@ -246,6 +246,35 @@ def reset_parameters(self):
             ):
                 m.reset_parameters()
 
+    def validate_reset_parameters(self):
```

Comment:
It is likely not needed but it might be safer to add:

`with torch.no_grad():`

Reply:
True, and true